src/main/java/com/codeborne/selenide/impl/Navigator.java

```diff
@@ -35,14 +45,32 @@ protected String absoluteUrl(String relativeUrl) {
   }
 
   protected void navigateToAbsoluteUrl(String url) {
+    navigateToAbsoluteUrl(url, "", "", "");
+  }
+
+  protected void navigateToAbsoluteUrl(String url, String domain, String login, String password) {
     if (isIE() && !isLocalFile(url)) {
       url = makeUniqueUrlToAvoidIECaching(url, System.nanoTime());
+      if (!domain.isEmpty()) domain += "\\";
+    }
+    else {
+      if (!domain.isEmpty()) domain += "%5C";
+      if (!login.isEmpty()) login += ":";
+      if (!password.isEmpty()) password += "@";
+      int idx1 = url.indexOf("//") + 2;
+      url = url.substring(0, idx1 - 2)
+              + "//"
+              + domain
+              + login
+              + password
+              + url.substring(idx1);
     }
 
     SelenideLog log = SelenideLogger.beginStep("open", url);
     try {
       WebDriver webdriver = getAndCheckWebDriver();
       webdriver.navigate().to(url);
+      if (isIE()) Selenide.switchTo().alert().authenticateUsing(new UserAndPassword(domain + login, password));
```

Comment:
Hey Guys,
in case if BasicAuth is not required then exception will be thrown since we will not be able to switch to alert.
We should handle this case inside an additional try-catch block (without exception throwing) or introduce the new Configuration property